src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfirm.tsx

```diff
@@ -86,9 +153,17 @@ export const LimitOrderConfirm = () => {
               </Row.Label>
               <Row.Value textAlign='right'>
                 <HStack>
-                  <Amount.Crypto value={'0.002134'} symbol={'WETH'} />
+                  {/*
+                    TODO: the rate differs from the input page because we're using the quoted values
+                    here instead of the user input. We need to decide how to handle this because the
+                    quote is likely what gets executed.
+                  */}
```

Comment:
q: I guess it's fine now? On the trade, we already display the rate at input time vs quote at confirmed time, we might need to explain to the user that those values are the "final" values thought

Reply:
In this case the difference is large, so I'll need to follow up with a solution. It looks like a quirk of the cowswap API, because when executing they recommend to override the quote amounts with the original input

Reply:
Feel free to resolve at any time